### ISSUE TITLE
overridden?

### ISSUE THREAD

**Issue by steelgriffin1** [bitfocus/companion-module-uts-remotevolume#1]
this is the error I get from the log in companion:
uts-remotevolume@1.0.0: UTS: Remote Volume (Overridden)

**Comment by ultimategmbh** [comment]
Do you have any local instance of the module installed?
But are you encountering any problems? Is anything not working - or do you receive this message but everything is working fine?

**Comment by steelgriffin1** [comment]
Yes I am testing the module on the windows machine that is running companion. The log shows that it found the module in the folder that I directed it to but the next line says “overridden” when it tries to activate the module. On Dec 2, 2024, at 3:15 PM, ultimategmbh ***@***.***> wrote:﻿
Do you have any local instance of the module installed?
But are you encountering any problems? Is anything not working - or do you receive this message but everything is working fine?

**Comment by Julusian** [comment]
'Overridden' simply means it is using a local/dev version of the module, not one that is distributed with companion.
So it doesn't sound like anything is wrong here
